.drone.star

```diff
@@ -1174,6 +1174,8 @@ def ocisService():
 			'REVA_STORAGE_OWNCLOUD_REDIS_ADDR': 'redis:6379',
 			'REVA_OIDC_ISSUER': 'https://ocis:9200',
 			'REVA_STORAGE_OC_DATA_SERVER_URL': 'http://ocis:9164/data',
+			'REVA_STORAGE_HOME_EXPOSE_DATA_SERVER': 1,
+			'REVA_STORAGE_OC_EXPOSE_DATA_SERVER': 1,
```

Comment:
Looks like something changed in ocis/reva since last time and update doesn't work without these configs.